packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -52,6 +52,7 @@ export interface IDetailsRowProps extends React.Props<DetailsRow> {
   getRowAriaDescribedBy?: (item: any) => string;
   checkButtonAriaLabel?: string;
   checkboxCellClassName?: string;
+  customRowRenderer?: (props: IDetailsRowFieldsProps) => JSX.Element;
```

Comment:
`rowFieldsAs: 
  React.StatelessComponent<IDetailsRowFieldsProps> | 
  React.ComponentClass<IDetailsRowFieldsProps>;
`

for consistency, can you name this `rowFieldsAs`?

Then, in your destructor of the props:

```tsx
const {
  rowFieldsAs: RowFields = DetailsRowFields
};
```
This will alias the prop as "RowFields" and default it to DetailsRowFields if not provided.

Then, you can just use it:

```tsx
<RowFields
   ...stuff
/>
```

Reply:
OK... Done